backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -4825,3 +4825,14 @@ databaseChangeLog:
           - sql:
               sql: |
                 DROP TYPE ${database.defaultSchemaName}.TEST_RESULT_UPLOAD_ERROR;
+  - changeSet:
+      id: result_upload_error-add-no-phi-user
+      author: nathan@skylight.digital
+      comment: Adds permission for the no-PHI user to select from the bulk test result upload errors
+      changes:
+        - tagDatabase:
+            tag:results_upload_error-add-no-phi-user
```

Comment:
can you add a space here

Reply:
Keen eye! 🕵️ added in [212c8d1](https://github.com/CDCgov/prime-simplereport/pull/6155/commits/212c8d1aa174f4f387e97137e116cd84600ec900).